pkg/nfd-worker/nfd-worker.go

```diff
@@ -508,6 +508,7 @@ func createFeatureLabels(sources []source.FeatureSource, labelWhiteList regexp.R
 	labels = Labels{}
 
 	// Do feature discovery from all configured sources.
+	klog.Info("doing feature discovery...")
```

Comment:
Ummm
starting feature discovery? or something better than `doing`? just a thought

Reply:
Mm, possibly not the best wording. Changed to *starting* as suggested and added an accompaniying *completed* log message after feature discovery has been finished

Reply:
Looks good to me, thanks!